live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/transform/response/TransformResponse.java

```diff
@@ -37,5 +38,8 @@
 @Validated
 public record TransformResponse(
         @NotBlank String targetReference,
-        @NotNull @JsonDeserialize(using = ClientDataDeserializer.class) ClientData clientData)
+        @NotNull @JsonDeserialize(using = ClientDataDeserializer.class) ClientData clientData,
+        @Positive int status,
+        String errorDetails)
```

Comment:
Is this field used anywhere? Maybe it could be passed also to the `TransforamtionFailedException` ?

Reply:
I changed the approach in a second pr: https://github.com/Alfresco/hxinsight-connector/pull/254 so I would leave it as it'll be changed there anyway